dev/ci/README-developers.md

```diff
@@ -42,9 +42,34 @@ See also [`test-suite/README.md`](../../test-suite/README.md) for information ab
 
 ### Breaking changes
 
-When your PR breaks an external project we test in our CI, you must
-prepare a patch (or ask someone—possibly the project author—to
-prepare a patch) to fix the project. There is experimental support for
+When your PR breaks external projects we test in our CI, you must
+assess the breakage. If it is a bug your PR introduces, it should be
+fixed. Otherwise, you must assess the impact of the breakage on a few
+CI entries at the bottom of its hierarchy, typically mathcomp, stdlib
+and stdpp. For legitimate breakages, for instance to remove something
+already deprecated, you must then write porting instructions (typically
+the PR changelog), based on your previous assesment.
+The PR assignee will finally ask the project maintainers to
+prepare a patch. For plugins, you must prepare yourself an overlay
+(you can ask the CI project maintainer for help).
+Ultimately, for non-plugin CI projects, the responsibility of
+preparing a patch falls on the project maintainer, but PR writers are
+encouraged to be helpful and may prepare patches themselves to
+facilitate a smooth process of adaptation.
+To ask project maintainers to adapt their development, the PR assignee
+can use the following template
```

Comment:
This looks good, but since this is long and it describes a well-defined process, I would encourage the use of bullet-points here.

Furthermore, this doesn't describe what "legitimate breakage" means, besides giving an example of the most obvious case. But many things beyond removing an already deprecated feature can be legitimate (and OTOH removing a recently deprecated but highly used feature might still be debatable).

So I would suggest clarifying that what counts as a legitimate breakage is something to be decided collectively (author, reviewers, assignee) based on the preliminary porting assessment made by the author.

Reply:
I agree, I gave it an attempt following your directions.